src/actions/publish.js

```diff
@@ -112,10 +144,12 @@ export function attemptToPublish(): ThunkAction<Promise<boolean>> {
 
       // Upload the profile, and notify it with the amount of data that has been
       // uploaded.
-      const hash = await startUpload(gzipData, uploadProgress => {
+      const hashOrToken = await startUpload(gzipData, uploadProgress => {
         dispatch(updateUploadProgress(uploadProgress));
       });
 
```

Comment:
In the future, this is where we'll dispatch an action to store the token to a local storage space.